Tests/interop/net/test_loadorder.py

```diff
@@ -1,125 +0,0 @@
-#####################################################################################
```

Comment:
It was just calling the other loadorder tests, which will be run automatically now.